pkgs/development/python-modules/cylp/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+, python
+, pytestCheckHook
+, cython
+, cbc
+, bzip2
+, zlib
+, pkg-config
```

Comment:
Personal preference is to sort these in some order: either in usage order below, or alphabetical.

Reply:
I sorted by usage order.